cranelift/filetests/filetests/runtests/simd-select.clif

```diff
@@ -0,0 +1,66 @@
+test interpret
+test run
+target aarch64
+target s390x
+target x86_64 ssse3 has_sse41=false
```

Comment:
This may be a bit of a dated copy/paste now that SSSE3 is the baseline for x86_64 simd support, but I've only been doing this in tests that exercise post-SSE2 instructions which I'm not sure if `select` does, but basically it's ok to delete this line if you'd like